app/views/providers/check_benefits/index.html.erb

```diff
@@ -1,31 +1,39 @@
-<%= content_for(:panel) do %>
-  <div class="govuk-panel govuk-panel--confirmation">
-    <h2 class="govuk-panel__title">
+<div class="interruption-panel">
+  <%= page_template page_title: '', column_width: 'full'  do %>
+<%
+# blank title - title dealt with below
```

Comment:
`page_template` passes `page_title` to `content_for(:page_title)`. This block has two main functions:

1. It is used in to populate the return from the helper method `page_heading`, which itself is used in the _default_ page template to add an `<h1>` tag at the top of the page.
2. It populates the page title in the page layout: `HTML <head> <title>`

By passing an empty string into "page_title" you've effectively removed the unique HTML page title from this page.

If you want to alter where the page heading is displayed on the page, use the _basic_ template, and then insert `page_heading` at the location where you want it. 

I'm not quite sure what your main heading is. I think it is `@legal_aid_application.applicant.full_name + t("#{result_namespace}.title")`

However, that's not a good way to use a translation. In other languages, the sentence may not start with the person's name. For (an obscure example) "Fred Blogs must complete a financial assessment" in Cebuano is "Ang Fred Blogs kinahanglang makompleto ang pinansyal nga pagtasa". So it better to pass the name into the translation, so that it can be placed into the correct part of the sentence. The translation code becomes:
```
title: "%{name} must complete a financial assessment"
```
And it's used like this:
```erb
t("#{result_namespace}.title", name: @legal_aid_application.applicant.full_name)
```

So here:
```erb
<%= page_template(
          page_title: t("#{result_namespace}.title", name: @legal_aid_application.applicant.full_name), 
          column_width: 'full',
          template: :basic
        ) do %>
```
Then the `<h1>` on lines 6-9 is replaced with:
```erb
<%= page_heading(size: 'l') %>
```

Reply:
Thanks, will re-do.